modules/vsphere-problem-detector-metrics.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * storage/vsphere-problem-detector-operator.adoc
+
+:operator-name: vSphere Problem Detector Operator
+
+[id="vsphere-problem-detector-operator-metrics_{context}"]
+= Metrics for the vSphere problem detector
+
+The {operator-name} exposes the following metrics for use by the Prometheus-based {product-title} cluster monitoring stack.
+
+.Metrics exposed by the {operator-name}
+[cols="2a,8a",options="header"]
+|===
+|Name |Description
+
+|`vsphere_cluster_check_total`
+|Number of vSphere cluster-level checks that were performed by the problem detector. This count includes both successes and failures.
```

Comment:
Does "vSphere cluster-level checks" refer to checks for a vSphere cluster, or an OCP cluster running on vSphere? If it is the latter, it might be useful to be more explicit to avoid misinterpretation.

The same applies to the other descriptions in this module, but I won't highlight the others.